website/docs/docs/build/metricflow-commands.md

```diff
@@ -222,10 +221,8 @@ The list of available saved queries:
 
 The following command performs validations against the defined semantic model configurations.
 
-Note, in dbt Cloud you don't need to validate the Semantic Layer config separately. Running a dbt command (such as `dbt parse`, `dbt build`, `dbt compile`, `dbt run`) automatically checks it.
-
 ```bash
-
+dbt sl validate-configs # dbt Cloud users
```

Comment:
same thing here, should be just `dbt sl validate`, ``validate-config` is used only in the MF CLI, which would be `mf validate-config`, we are looking to rename that to be just `mf validate`, it just hasn't been done yet

Reply:
fixed in https://github.com/dbt-labs/docs.getdbt.com/pull/5930/commits/51a821ce13b69866bcf690b9f3761312725fef27